src/java.net.http/share/classes/jdk/internal/net/http/Http1HeaderParser.java

```diff
@@ -116,15 +116,16 @@ boolean parse(ByteBuffer input) throws ProtocolException {
 
         while (canContinueParsing(input)) {
             switch (state) {
-                case INITIAL -> state = State.STATUS_LINE;
-                case STATUS_LINE -> readResumeStatusLine(input);
-                case STATUS_LINE_FOUND_CR, STATUS_LINE_FOUND_LF -> readStatusLineFeed(input);
-                case STATUS_LINE_END -> maybeStartHeaders(input);
-                case STATUS_LINE_END_CR, STATUS_LINE_END_LF -> maybeEndHeaders(input);
-                case HEADER -> readResumeHeader(input);
-                case HEADER_FOUND_CR, HEADER_FOUND_LF -> resumeOrLF(input);
-                case HEADER_FOUND_CR_LF -> resumeOrSecondCR(input);
-                case HEADER_FOUND_CR_LF_CR -> resumeOrEndHeaders(input);
+                case INITIAL ->                                     state = State.STATUS_LINE;
```

Comment:
Looks good. Although, I think you can improve it further if you align the lambda operators as well

Reply:
Yes right. Just noticed I had missed that. Fixed in latest commit :)

Reply:
Looks much better, Kartik. Thanks